twistr/events.go

```diff
@@ -597,6 +727,22 @@ func PlayMissileEnvy(s *State, player Aff) {
 	   immediately. If it contains an opponent’s Event, use the Operations value
 	   (no Event). The opponent must use this card for Operations during their next
 	   action round. */
+	s.TurnEvents[MissileEnvy] = player
+	maxOps := 0
+	for _, c := range s.Hands[player.Opp()].Cards {
+		if c.Ops > maxOps {
+			maxOps = c.Ops
+		}
+	}
+
+	selected := SelectCard(s, player.Opp(), ExceedsOps(maxOps-1))
+	if selected.Aff == player || selected.Aff == NEU {
+		PlayEvent(s, player, selected)
+	} else {
+		PlayOps(s, player, selected)
```

Comment:
PlayOps allows the opponent to implement her own event. This should call into ConductOps, which does not include that aspect.